pytorch/Dockerfile

```diff
@@ -80,26 +80,43 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missin
     gcc \
     libgl1-mesa-glx \
     libglib2.0-0 \
-    virtualenv && \
-    apt-get clean && \
-    rm -rf /var/lib/apt/lists/*
+    virtualenv
 
 ENV SIGOPT_PROJECT=.
 
 WORKDIR /
 COPY multinode-requirements.txt .
+COPY generate_ssh_keys.sh .
```

Comment:
For now, I'm just going to copy the existing one in `workflows/charts/huggingface-llm` because that one will eventually be deleted, effectively moving it over here.